platform-sdk/swirlds-config-api/build.gradle.kts

```diff
@@ -15,13 +15,17 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.maven-publish")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
+    id("com.swirlds.platform.maven-publish")
+    id("org.gradle.java-test-fixtures")
 }
 
 dependencies {
-  // Test Dependencies
-  testImplementation(testLibs.bundles.junit)
-  compileOnly(libs.spotbugs.annotations)
+    // Test Dependencies
+    testImplementation(testLibs.bundles.junit)
```

Comment:
intentional?

Reply:
Looks like spotless did it since it was changed for the first time?